### DISCUSSION TITLE
白名单改个名会不会好点？

### DISCUSSION THREAD

**Discussion by Miuzarte** [Mikotwa/FuckLocation#24] [Ideas]
想发discussion但是没有 就直接发issues了
个人认为白名单是允许其中的应用获取真实位置，感觉现在这样的逻辑有点问题
觉得这个意见不好直接close就行（

**Comment by Mikotwa** [comment]
> 想发 discussion 但是没有 就直接发 issues 了

感谢提醒！本仓库将开放 discussion 作为讨论渠道之一。

关于“白名单”这个名称，我自己的原初想法是取材自其名字的”少数性”，意指“针对特定应用的客制化名单”。
当时取名的时候，确实没想到会带来现在的误解。

欢迎提议更好的替代名称。

**Comment by Mikotwa** [comment]
该 issue 已经转换为 discussion。欢迎讨论

**Comment by Miuzarte** [comment]
### Fuck(ed) list

**Reply by Mikotwa** [reply]
取名鬼才，草

**Comment by gyq180** [comment]
目标应用

**Comment by AdminHcat** [comment]
作用域
